playpen/all_26.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/bash -e
+
+#  When using this script, you will need to fix your database accordingly. If moving backward, you
+#  will need to drop the database, and you will always need to run the migrations.
+
+# You need the .bashrc that the Vagrant or dev-setup.sh environment creates. You can get it from
+# playpen/ansible/roles/dev/files/bashrc if you aren't using one of those envs.
+. ~/.bashrc
+
+# Switch back to pulp-2.6
+pstop
+
+pushd ~/devel
+for r in {pulp_deb,pulp_docker,pulp_openstack,pulp_ostree,pulp_puppet,pulp_python,pulp_rpm,pulp}; do
+    if [ -d $r ]; then
+        pushd $r
+        sudo ./pulp-dev.py --uninstall
+        popd
+    fi
+done
+
+pushd pulp
+git checkout 2.6-dev
+find ~/devel -name "*.py[c0]" -delete
+sudo ./pulp-dev.py --install
+popd # pulp
+
+# This one is necessary to work around a Pulp httpd config bug, where some plugin MUST install an
+# httpd config.
+pushd pulp_rpm
+git checkout 2.6-dev
+find ~/devel -name "*.py[c0]" -delete
+sudo ./pulp-dev.py --install
+popd # pulp_rpm
+
```

Comment:
Should this script also go checkout the 2.6-dev branches in each of the other repositories, such as `pulp_puppet`? When I run this script as is, it leaves at least that repository alone and on the master branch, which of couse causes breakage when I try to use the rolled-back system.